lib/libcheri/libcheri_sandbox_methods.c

```diff
@@ -87,6 +87,7 @@ struct sandbox_provided_classes {
 	size_t				  spcs_maxclasses; /* Array size */
 	size_t				  spcs_nmethods; /* Total methods */
 	vm_offset_t			  spcs_base;	/* Base of vtable */
+	size_t				  spcs_size;	/* Size of section */
```

Comment:
Only ever seems to be written to?

Reply:
Yes, I originally added it for debugging and thought I'd need it. Probably fine to remove it.